schema/vrs.json

```diff
@@ -21,7 +21,7 @@
          }
       },
       "MolecularVariation": {
-         "description": "A variation on a contiguous molecule.",
+         "description": "A :ref:`variation` on a contiguous molecule.",
```

Comment:
@ahwagner while i think this is cool. I'm just wondering how description and other attribute string values that contain RST tags will impact other tools that use these fields to display the schema documentation (i.e. swagger). I may be missing the mark here and it really may not be a big deal, but are we thinking about how other commonly used tools will work with our enhanced RestructuredText syntax. I suppose the big value of RST and markdown in general is that it is still very legible even in an unrendered state.

Reply:
I think this is good feedback, will address.